conf/telemetry/steering_rover.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<!DOCTYPE telemetry SYSTEM "telemetry.dtd">
+<telemetry>
+
+
+  <process name="Main">
+
+    <mode name="default" key_press="d">
+      <message name="AUTOPILOT_VERSION"      period="11.1"/>
+      <message name="ROVER_STATUS"           period="1.2"/>
+      <message name="ROTORCRAFT_FP"          period="0.25"/>
+      <message name="STEERING_ROVER_DATA"    period="0.5"/>
```

Comment:
maybe a new telemetry file is not needed and can be merged with default_rover.xml